runtime/databricks/automl_runtime/forecast/prophet/model.py

```diff
@@ -262,7 +264,12 @@ def predict(self, context: mlflow.pyfunc.model.PythonModelContext, model_input:
         test_df = model_input.copy()
         test_df["ts_id"] = test_df[self._id_cols].astype(str).agg('-'.join, axis=1)
         test_df.rename(columns={self._time_col: "ds"}, inplace=True)
-        predict_df = test_df.groupby("ts_id").apply(lambda df: self.model(df.name[0]).predict(df)).reset_index()
+
+        def model_prediction(df):
+            model = self.model(df.name)
+            if model:
+                return model.predict(df)
+        predict_df = test_df.groupby("ts_id").apply(lambda df: model_prediction(df)).reset_index()
```

Comment:
What would the `apply` do if the `model_prediction` returns `None` (not in the `if` branch)?

Reply:
It is fine in that case, let me add a unit test for this case.

Reply:
Confirmed in the unit test, it will return nan.